README.md

```diff
@@ -15,28 +15,129 @@ This is a promise based interface for interacting with a source control manageme
 ### configure
 The `configure` function takes in an object and resets the configuration values
 
-### formatScmUrl
+### parseUrl
 Required parameters:
 
 | Parameter        | Type  |  Description |
 | :-------------   | :---- | :-------------|
-| scmUrl        | String | Scm Url to format |
+| config             | Object | Configuration Object |
+| config.checkoutUrl | String | Checkout url for a repo to parse |
+| config.token  | String | The github token to check permissions on |
```

Comment:
Haven't implemented this for bitbucket yet, but it's not always going to be `github token`.